atomic_reactor/inner.py

```diff
@@ -404,6 +597,170 @@ def _update(data):
         return new_data
 
 
+@dataclass
+class ImageBuildWorkflowData(ISerializer):
+    """Manage workflow data.
+
+    Workflow data is those data which are generated values by plugins through
+    the whole build workflow (pipeline) and must be shared in every build tasks.
+
+    These data can be dumped into dictionary object in order to be saved into a
+    file as JSON data, and then be loaded while atomic-reactor launches again to
+    execute another set of plugins for a different build task.
+    """
+
+    dockerfile_images: DockerfileImages = field(default_factory=DockerfileImages)
+    tag_conf: TagConf = field(default_factory=TagConf)
```

Comment:
since this TagConf will be for the final image (manifest list)

do we need also TagConf in results for arch specific unique images ?

Reply:
You would have to ask @mkosiarc what exactly his plans are, but IIUC arch-specific tags will be a new addition to the TagConf class and/or the tag_from_config plugin result. Either way, this PR shouldn't have to care about that